Faraday_Proxy_Tools/FaradayIO/commandmodule.py

```diff
@@ -131,6 +131,55 @@ def create_fixed_length_packet(data, fixed_legth):
         return padded_data
 
 
+def create_fixed_length_packet_padding(data, fixed_legth, padding_byte):
+    """
+    A simple function that accepts a string of databytes and appends padding bytes up to a fixed length. This version
+    of the packing routine allows for the selection of a specific padding byte.
+
+    :param data: The string of bytes that will be appdended padding bytes to create the fixed length packet
+    :param fixed_length: The length of the fixed length packet in bytes
+    :param padding_byte: The byte to use as padding
+
+    :return: Returns the "data" padded with additional bytes in the fixed sized specified.
+
+    :Example:
+
+        >>> pack_data("Test data message", 20, 0x30)
+        'Test data message\x00\x00\x00'
```

Comment:
I updated to show a better example for all 3 padding functions:

![image](https://cloud.githubusercontent.com/assets/339144/23091456/7b59e736-f56b-11e6-8575-86c59610874d.png)

![image](https://cloud.githubusercontent.com/assets/339144/23091457/82fbbf96-f56b-11e6-80b4-c5a657ce601d.png)

![image](https://cloud.githubusercontent.com/assets/339144/23091460/8a7d034c-f56b-11e6-85ed-39b758acdf93.png)